### PR TITLE
Fixes #12852: Document rudder branding plugin

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/12852

### CODE REVIEW

File: 80_extension_and_integration/20_Rudder_plugins.txt [hunk 1]
```diff
@@ -76,3 +76,44 @@ You also can interact with Rudder developers through the
 https://www.rudder-project.org/site/community/mailing-lists/[community] regular
 communication channels.
 
+
+==== Rudder Branding Plugin
+
+This plugin allows for changing the apparence of the login page, and the top bar of Rudder, by adding a colored bar with a text. Its main usage is to easily differentiate multiple Rudders server, and making sure the changes are made on the right environnement
```
Comment (amousset) [inline 80_extension_and_integration/20_Rudder_plugins.txt]: missing ending .

File: 80_extension_and_integration/20_Rudder_plugins.txt [hunk 2]
```diff
@@ -76,3 +76,44 @@ You also can interact with Rudder developers through the
...
+
+===== Prerequisite
+
+The rudder Branding plugin requires Rudder 4.3.3 or higher.
```
Comment (amousset) [inline 80_extension_and_integration/20_Rudder_plugins.txt]: Rudder

File: 80_extension_and_integration/20_Rudder_plugins.txt [hunk 3]
```diff
@@ -76,3 +76,44 @@ You also can interact with Rudder developers through the
...
+
+====== Customization
+
+A new page is available within Rudder, to customize the apparence. This page is in the Settings section, called Branding. It allows for setting the top Bar and the Login page, with a color and a text, and a real-time display of the expected result
+
+image:./images/branding/Configure.png[Configure apparence]
+
+The text bar accepts any UTF-8 character or emojis, as displayed in this example
+
+image:./images/branding/Customised_bar.png[Customized bar]
```
Comment (amousset) [inline 80_extension_and_integration/20_Rudder_plugins.txt]: I must admit I checked if it was "Rudder 2000".

Comment (ncharles) [reply 80_extension_and_integration/20_Rudder_plugins.txt]: :)

Comment (ncharles) [reply 80_extension_and_integration/20_Rudder_plugins.txt]: image

**Comment by ncharles** [comment]
PR rebased

**Comment by ncharles** [comment]
Commit modified

**Comment by ncharles** [comment]
Commit modified

**Comment by ncharles** [comment]
Commit modified

**Pull request merged**